### PR TITLE
chore: add spell checker config

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/938/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/938/storybooks/a11y-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/938/storybooks/ag-grid-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/938/storybooks/core-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/938/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/938/storybooks/layout-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/938/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/938/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/938/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/938/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/938/storybooks/tabs-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/938/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/938/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/938/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/938/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/938/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/938/playground/)

File: .vscode/settings.json [hunk 1]
```diff
@@ -3,18 +3,5 @@
   "editor.formatOnSave": true,
   "prettier.requireConfig": true,
   "coverage-gutters.showLineCoverage": true,
-  "cSpell.words": [
-    "Autonumeric",
```
Comment (Blackbaud-SandhyaRajasabeson) [inline .vscode/settings.json:7]: Does capitalization matter? some of these are lowercase and some (which I'm assuming are proper nouns) are not.

Comment (Blackbaud-TrevorBurch) [reply .vscode/settings.json:7]: No. `cspell` is case insensitive by default in English. 

https://cspell.org/docs/case-sensitive/

Comment (Blackbaud-SandhyaRajasabeson) [reply .vscode/settings.json:7]: should we make everything lowercase then just as a standard for these lists so theres no confusion in the future if someone wants to add something?

Comment (Blackbaud-TrevorBurch) [reply .vscode/settings.json:7]: I can. The VSCode extension’s quick fix action adds things as they were in that location. Which is how some of the capitalized ones got added.

Comment (Blackbaud-SandhyaRajasabeson) [reply .vscode/settings.json:7]: ooooh that will probably be an easier way for ppl to add it in the future than manually typing things in. So i guess its a better practice to just leave it be.

Comment (Blackbaud-TrevorBurch) [reply .vscode/settings.json:7]: I went ahead and updated the current ignores (I hadn’t seen your last message). We don’t have to worry about it moving forward though.

**Review by Blackbaud-SandhyaRajasabeson — APPROVED** [review]

**Pull request merged**